pkg/resource/obtenantoperation_task.go

```diff
@@ -0,0 +1,314 @@
+/*
+Copyright (c) 2023 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+
+package resource
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
+
+	"github.com/oceanbase/ob-operator/api/constants"
+	"github.com/oceanbase/ob-operator/api/v1alpha1"
+	"github.com/oceanbase/ob-operator/pkg/oceanbase/operation"
+	"github.com/oceanbase/ob-operator/pkg/oceanbase/param"
+)
+
+func (m *ObTenantOperationManager) ChangeTenantRootPassword() error {
+	con, err := m.getTenantSysClient(m.Resource.Spec.ChangePwd.Tenant)
```

Comment:
getTenantClient?
Sys is the default tenant name created when obcluster bootstraped

Reply:
Actually I mean getting root client here, I made a mistake here. I will change it to `getTenantRootClient`